solutions/USACO dec15-breed-counting.mdx

```diff
@@ -47,4 +47,59 @@ int main() {
 ```
 
 </CPPSection>
+<JavaSection>
+
+### Java Implementation
```

Comment:
analysis already contains java impl, don't see the point of having another

Reply:
Ah, mine was just a bit simpler.

Reply:
That's debatable, but ig it doesn't hurt to have another impl